web/src/components/molecules/Common/WorkspaceMenu/index.tsx

```diff
@@ -43,13 +33,43 @@ const WorkspaceMenu: React.FC<Props> = ({
   defaultSelectedKeys,
 }) => {
   const navigate = useNavigate();
-  const items: ItemType[] = [
-    getItem("Member", "member", <UsergroupAddOutlined />, "notPersonal"),
-    getItem("Account", "account", <UsergroupAddOutlined />, "personal"),
-    getItem("Integration", "integration", <ApiOutlined />, "both"),
-    getItem("Role", "role", <UserSwitchOutlined />, "notPersonal"),
-    getItem("API key", "api-key", <SearchOutlined />, "both"),
-    getItem("Settings", "settings", <SettingOutlined />, "notPersonal"),
+  const items: WorkspaceItemType[] = [
+    {
+      label: "Member",
+      key: "member",
+      icon: <UsergroupAddOutlined />,
+      show: "notPersonal" as MenuShowType,
```

Comment:
Is the `as MenuShowType` actually necessary to avoid type errors? I feel like as long as the value is one of the three string values in MenuShowType it shouldn't cause an error, but maybe im wrong?

Reply:
Typescript is showing type errors, I couldn't solve it only by casting like this.
I'll check this later as well because I'll get back to this menu and it will be linked to other pages.